### PR TITLE
Fixed setup script

### LINKED ISSUE #2493: Fix minor text issues with the Setup Script
**Describe the bug**

1. The some of the setup script prompts need better clarification 

**To Reproduce**

1. Run the setup script
2. Note the requested changes below
3. The test code coverage must be maintained.
4. All tests must be valid and pass 

**Expected behavior**

- The suggested changes are applied

**Actual behavior**

- See below

**Screenshots**

1. Change to `Enter the Talawa-API server port (4000)`
    image
1. Change `Do you want to import sample data? (y/N)` to `Do you want to import Talawa sample data for testing and evaluation purposes? (y/N)`  
image
1. Change `Do you want to import default data?` to `Do you want to import the required default data to start using Talawa in a production environment?`
1. In both cases below change `Would you like to generate ` to `Would you like to us to auto-generate ... for you`.
    image

**Additional details**

- N/A

**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by ARYANSHAH1567**
The some of the setup script prompts need better clarification. So i fixed the setup script as per required.

**Issue Number:**

Fixes #2493 

## Summary by CodeRabbit

- **New Features**
	- Improved user prompts for clarity, including specific mentions of "auto-generate" and "Talawa" for better user guidance during token generation and data import.
	- Streamlined error handling in the application to enhance user experience.
- **Bug Fixes**
	- Addressed ESLint warnings related to unused variables to maintain code quality.

### CODE REVIEW

File: setup.ts [hunk 1]
```diff
@@ -219,6 +219,7 @@ export async function wipeExistingData(url: string): Promise<void> {
       }
       console.log("All existing data has been deleted.");
     }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (palisadoes) [inline setup.ts:222]: Remove the linting disable. Find an acceptable solution. We have a PR test to make sure they are not used.

File: setup.ts [hunk 2]
```diff
@@ -246,6 +247,7 @@ export async function checkDb(url: string): Promise<boolean> {
     } else {
       dbEmpty = true;
     }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (palisadoes) [inline setup.ts:250]: Remove the linting disable. Find an acceptable solution. We have a PR test to make sure they are not used.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Remove the linting disable. Find an acceptable solution. We have a PR test to make sure they are not used.

**Comment by ARYANSHAH1567** [comment]
> Remove the linting disable. Find an acceptable solution. We have a PR test to make sure they are not used.

@palisadoes found an alternate solution for it and fixed it. Kindly please review the changes

**Review by palisadoes — CHANGES_REQUESTED** [review]
1. When I run the script, I get this error.
    image
1. It may be due to this recently merged PR. If it is, it should be an easy fix.
     1. https://github.com/PalisadoesFoundation/talawa-api/pull/2509
1. Merge your code with the latest upstream 
2. Work with @ARYANSHAH1567 if necessary

**Review by palisadoes — APPROVED** [review]

**Pull request merged**